content/docs/pipelines/pipelines-deploy-alternatives.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "Pipelines Deploy Alternatives"
```

Comment:
Recommendation: Change the file name too. For example, change `/pipelines/pipelines-deploy-alternatives` to `/pipelines/deployment-alternatives`.

Reply:
@sarahmaddox Thanks for the suggestion.

`alternatives` is because we will soon add documentation for another alternative: GCP marketplace. So I'm just trying to avoid a rename of this piece of document in a short period. What do you think? or maybe we should just rename when we add that?

I will update to `Pipelines Deployment Alternatives` for now and change the file name as you suggested.